### PR TITLE
Helm v3 updates

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

- Helm v3 going from Beta to GA
- Fixed K8s links that were linking to specific, old versions

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203247468824199

### CODE REVIEW

File: docs/integrations/saas-cloud/istio.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ namespace=istio-system cluster={{cluster}}
 
 ## Collecting Logs and Metrics for the Istio App
 
-This section provides instructions for collecting logs and metrics for the Sumo App for Istio. Logs and metrics are collected with the Sumo Logic Helm chart. Istio [sample metrics](/#Sample-Metrics) and [sample log messages](#Sample-Log-Messages) are also provided, along with a [query sample](#Query-Sample).
+This section provides instructions for collecting logs and metrics for the Sumo App for Istio. Logs and metrics are collected with the Sumo Logic Helm chart. Istio [sample metrics](#Sample-Metrics) and [sample log messages](#Sample-Log-Messages) are also provided, along with a [query sample](#Query-Sample).
 
 This app supports Istio versions 1.8.x and 1.9.x+.
 
-The minimum version of Sumo Logic K8s Collection required is [V2.1.6](https://github.com/SumoLogic/sumologic-kubernetes-collection/releases/tag/v2.1.6).
+The minimum version of Sumo Logic K8s Collection required is [v2.1.6](https://github.com/SumoLogic/sumologic-kubernetes-collection/releases/tag/v2.1.6).
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/istio.md:46]: @mat-rumian should this still be 2.1.6?

Comment (swiatekm-sumo) [reply docs/integrations/saas-cloud/istio.md:46]: I think any supported version is fine, which currently means at least [`v2.11.0`](https://github.com/SumoLogic/sumologic-kubernetes-collection#documentation). So we can just remove this line. WDYT @perk-sumo ?

File: docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/install-telegraf.md [hunk 2]
```diff
@@ -158,9 +158,7 @@ If you're seeing the following error in Microsoft’s Services Control Panel App
 
 This section documents the steps for setting up Telegraf in a Kubernetes environment. Due to the dynamic nature of Kubernetes, we use the Telegraf Operator. 
 
-1. First you need to set up Sumo Logic’s Kubernetes collection.
-
-   * If you have not set up Sumo Logic’s Kubernetes collection, perform [these steps to set up Kubernetes collection](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/release-v1.2/deploy/docs/Installation_with_Helm.md#installation-with-helm).
+1. First you need to set up Sumo Logic’s Kubernetes collection. If you haven't, go to [Installation with Helm](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation-with-helm.md) and perform those setup steps.
```
Comment (swiatekm-sumo) [inline docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/install-telegraf.md:161]: Should be
```suggestion
1. First you need to set up Sumo Logic’s Kubernetes collection. If you haven't, go to [Installation with Helm](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation.md) and perform those setup steps.
```
And we should also make this change everywhere else it needs to be made.

File: docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md [hunk 3]
```diff
@@ -73,9 +73,9 @@ The [Sumo Logic Helm chart](https://github.com/SumoLogic/sumologic-kubernetes-co
...
 
-First step, set up Kubernetes collection with Helm, as described on https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation-with-helm.md.
+First step, [set up Kubernetes collection with Helm](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation-with-helm.md).
```
Comment (swiatekm-sumo) [inline docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md:78]:
```suggestion
First step, [set up Kubernetes collection with Helm](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation.md).
```

**Review by swiatekm-sumo — COMMENTED** [review]
LGTM, some minor comments.

Incidentally, should we add [markdown-link-check](https://github.com/gaurav-nelson/github-action-markdown-link-check) to the CI? It would automatically protect against broken links in the md documents.

File: docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/install-telegraf.md (hunk 2 above)
Comment (kimsauce) [reply docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/install-telegraf.md:161]: Thank you! Done.

File: docs/integrations/saas-cloud/istio.md (hunk 1 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/istio.md:46]: Thanks! I'll remove it for now just to get the docs out the door. @perk-sumo , if you had any other input, lemme know.

**Review by stacykor — APPROVED** [review]
LGTM.

**Pull request merged**